<ui version="4.0" >
 <author>José Manuel Díez Botella y Iván González Millán</author>
 <class>buscasubcuenta</class>
 <widget class="QDialog" name="buscasubcuenta" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Búsqueda de SUBCUENTA</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Selección</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="ComienceRadioButton" >
          <property name="text" >
           <string>Que comience por ...</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="ContengaRadioButton" >
          <property name="text" >
           <string>Que contenga ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>SUBCUENTA:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SubcuentaLineEdit" >
         <property name="maximumSize" >
          <size>
           <width>150</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>TÍTULO:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TituloLineEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTableView" name="tableView" />
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AceptarBoton" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-42.png</normaloff>:/images/librok-42.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelarBoton" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-7.png</normaloff>:/images/librok-7.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>TituloLineEdit</tabstop>
  <tabstop>SubcuentaLineEdit</tabstop>
  <tabstop>ComienceRadioButton</tabstop>
  <tabstop>ContengaRadioButton</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>AceptarBoton</tabstop>
  <tabstop>CancelarBoton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>AceptarBoton</sender>
   <signal>clicked()</signal>
   <receiver>buscasubcuenta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>348</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel" >
     <x>47</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelarBoton</sender>
   <signal>clicked()</signal>
   <receiver>buscasubcuenta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>499</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
